pkg/config/config.go

```diff
@@ -1056,6 +1062,13 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		return err
 	}
 
+	if c.Timezone != "" && !strings.EqualFold(c.Timezone, "local") {
+		_, err := time.LoadLocation(c.Timezone)
+		if err != nil {
+			return fmt.Errorf("invalid timezone %s", c.Timezone)
```

Comment:
Just a nit (also, for consistency with other errors):

```suggestion
			return fmt.Errorf("invalid timezone: %s", c.Timezone)
```